components/xip-patch/xip-patch.sh

```diff
@@ -56,12 +58,23 @@ generateCerts() {
 }
 
 createOverridesConfigMap() {
-    if [ -z "$(kubectl get configmap -n kyma-installer net-global-overrides --ignore-not-found)" ]; then
+    if [[ "$GARDENER_ENVIRONMENT" == true ]]; then
         kubectl create configmap net-global-overrides \
-            --from-literal global.ingress.domainName="$INGRESS_DOMAIN" \
-            --from-literal global.ingress.tlsCrt="$INGRESS_TLS_CERT" \
-            --from-literal global.ingress.tlsKey="$INGRESS_TLS_KEY" \
-            -n kyma-installer
+        --from-literal global.ingress.domainName="$INGRESS_DOMAIN" \
```

Comment:
We can shorten this a bit by extracting common parameters to a variable and then execute like this: `kubectl create configmap --from-literal someKey=someValue $COMMON_PARAMS`